### PR TITLE
[4.0] Help Button Improved

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #22544  .

### Summary of Changes
Added```title``` attribute in ```layouts/joomla/toolbar/basic.php```
Also added audible notifications to help button to satisfy the accessibility rules

### Testing Instructions
Go to Content => Articles 
Hover on the Help button.

### Expected result

Also audible notifications for screen readers

### Actual result
No visible and audible notification

### Documentation Changes Required
None

### CODE REVIEW

**Comment by C-Lodder** [comment]
It's a shame screen readers don't automatically detect the `target` attribute

**Comment by TobsBobs** [comment]
I have tested this item :white_check_mark: successfully on f0684b047305763276f4521683c017b4dc42c2ea<hr />

Unbenannt


<sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24138">issues.joomla.org/tracker/joomla-cms/24138</a>.</sub>

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -488,6 +488,7 @@ JGLOBAL_NUMBER_CATEGORY_ITEMS_LABEL="Show Article Count"
 JGLOBAL_NUMBER_ITEMS_LIST_DESC="Default number of articles to list on a page."
 JGLOBAL_NUMBER_ITEMS_LIST_LABEL="# Articles to List"
 JGLOBAL_NO_MATCHING_RESULTS="No Matching Results"
+JGLOBAL_OPENS_IN_A_NEW_WINDOW="Opens in a new window"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Please alpha order

**Comment by hardik-codes** [comment]
Now have a look @brianteeman

**Comment by zwiastunsw** [comment]
Opening the help screen in a new window is visible to sighted users. The change of context is not surprising in this case. In particular, closing the help window will return to the place where the window has been opened. 
However, it is advisable to notify blind users about opening in new window. For this purpose it is enough to use text visible only for screen readers. 
Additionally: Simultaneous use of the title and text attribute only for screen readers causes screen readers to announce the same message twice (except for JAWS, if title attribute reading is not enabled). 

In my opinion: use only the `title` attribute **or** text visible only for screen readers. Personally, I am in favour of text that is only visible to screen readers.

**Comment by brianteeman** [comment]
The WCAG guidelines are clear that all users should be informed if a link opens in a new window
https://www.w3.org/TR/WCAG20-TECHS/G201.html

**Comment by astridx** [comment]
I think we can not do it like this, because `/libraries/src/Toolbar/Button/DropdownButton.php` uses this layout too and because of this some other buttons have this tooltip, too. (`/libraries/src/Toolbar/Button/DropdownButton.php` inherited it from `/libraries/src/Toolbar/Button/AbstractGroupButton.php`)
For example the button `Change Status` in `/administrator/index.php?option=com_content&view=articles`. See picture.

Bildschirmfoto von 2019-03-09 16-30-57

**Comment by zwiastunsw** [comment]
I do not want to discuss unnecessarily, but: 
2. [G201](https://www.w3.org/TR/WCAG20-TECHS/G201.html) is an advisory technique (recommended, but not obligatory). 
2. Success Criteria [3.2.1](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html) and [3.2.2](https://www.w3.org/WAI/WCAG21/Understanding/on-input) (Level A) relate to an automatic change of context when the light is focused or after data has been entered. This is not the case here. Here we have a situation which is the [Success criterion 3.2.5](https://www.w3.org/WAI/WCAG21/Understanding/change-on-request) (Level AAA). 
3. Since we have a change of context here which may cause confusion especially among blind people  **this PR is needed**. 
4. My intention was to advise **to use one way to notify the user** (title or text attribute for screen readers only). Using both ways causes unnecessary information noise. This is also a violation of accessibility.

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 805e2a4b231c1a84acfdbe17f75b8fe99a6c6a14<br /><br />the basic button is used for more than the help buttons. Possible option would be to have a new layout for the help button<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24138">issues.joomla.org/tracker/joomla-cms/24138</a>.</sub>

**Comment by hardik-codes** [comment]
@brianteeman is there any layout for the help button already ?

**Comment by brianteeman** [comment]
No, that's why I said new

**Comment by Quy** [comment]
Closing as not the proper fix.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24138">issues.joomla.org/tracker/joomla-cms/24138</a>.</sub>

**Pull request closed without merging**